Xamarin.Forms.Platform.Android/FastRenderers/ButtonRenderer.cs

```diff
@@ -315,12 +315,12 @@ void Initialize()
 			AddOnAttachStateChangeListener(this);
 			OnFocusChangeListener = this;
 
-			Tag = this; 
-        }
+			Tag = this;
+		}
 
 		void UpdateBitmap()
 		{
-			if (Element == null)
+			if (Element == null || _isDisposed)
```

Comment:
These methods are called all over this renderer, not just in OnPropertyChanged. The original PR that "fixed" this issue added it in just OnElementChanged. This seemed like the best way to prevent regressing this bug, and since it's a crasher, I wanted to be safe.